### PR TITLE
chore: initial commit of doc update

### PR DESCRIPTION
**Pull request by UnicornChance**
## Description
Adding docs for using, modifying, explaining the new Google IDP changes.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

File: docs/IDENTITY_PROIVIDER.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+              path: configImage
+            - name: GOOGLE_SSO_ENABLED
+              description: "Enable Google SSO IDP"
+              path: realmInitEnv.REALM_GOOGLE_IDP_ENABLED
+              default: true
+            - name: GOOGLE_SSO_CLIENT_ID
+              description: "Set Google SSO Client ID"
+              path: realmInitEnv.REALM_GOOGLE_IDP_CLIENTID
+              default: "{fill in value here}"
+            - name: GOOGLE_SSO_CLIENT_Secret
```
Comment (rjferguson21) [inline docs/IDENTITY_PROIVIDER.md:161]:
```suggestion
            - name: GOOGLE_SSO_CLIENT_SECRET
```

Comment (UnicornChance) [reply docs/IDENTITY_PROIVIDER.md]: just updated this to use the values like you used for testing. Easier to understand and less unnecessary fields/words.

File: docs/IDENTITY_PROIVIDER.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            "disableUserInfo": "false",
+            "sendIdTokenOnLogout": "true",
+            "passMaxAge": "false"
+          }
+        }
+    ],
+    ```
+    * note the `${REALM_GOOGLE_IDP_ENABLED}`, `${REALM_GOOGLE_IDP_CLIENT_SECRET}`, and `${REALM_GOOGLE_IDP_CLIENTID}` pieces that are provided by uds-core environment variables
+
+2. Update `uds-core`
```
Comment (rjferguson21) [inline docs/IDENTITY_PROIVIDER.md:96]: I think this part can be removed, and you should just skip to showing how the `realmInitEnv` is configured via the values.

Comment (rjferguson21) [inline docs/IDENTITY_PROIVIDER.md:96]: I think from this section you should remove both 1 and 3 and just show the `values.yaml`

File: docs/CUSTOMIZE.md [hunk 3]
```diff
@@ -75,6 +75,8 @@ The `UDS Identity` realm is defined in the realm.json found in [src/realm.json](
 > [!CAUTION]
 > Be aware that changing values in the realm may also need be to updated throughout the configuration of Keycloak and Authservice in `uds-core`. For example, changing the realm name will break a few different things within Keycloak unless those values are changed in `uds-core` as well.
 
+In addition environment variables can be configured through uds-core + bundle overrides. These environment variables will be created with a prefix `realm_`. Additional context can be found in [IDENTITY_PROVIDER.md](./IDENTITY_PROIVIDER.md#override-default-sso-values).
```
Comment (rjferguson21) [inline docs/CUSTOMIZE.md:78]:
```suggestion
In addition environment variables can be configured through uds-core + bundle overrides. These environment variables will be created with a prefix `REALM_`. Additional context can be found in [IDENTITY_PROVIDER.md](./IDENTITY_PROIVIDER.md#override-default-sso-values).
```

**Review by rjferguson21 — APPROVED** [review]

**Pull request merged**